projects/design-system-initiafy/src/lib/styles/help/visibility.scss

```diff
@@ -0,0 +1,3 @@
+.noVis {
```

Comment:
we should use dashes instead of camel case, google standard for writting css classes